csfieldguide/templates/interactives/rsa-no-padding.html

```diff
@@ -0,0 +1,47 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+	<div class="container">
+  	<div class="row">
```

Comment:
I think this repeated `row` and `col-12` elements, plus the `container` element are not needed. Might be worth checking.